### DISCUSSION TITLE
Set the trigger point when mouse moves to left to allow window to snap to left (where stage manager dock resides)

### DISCUSSION THREAD

**Discussion by thloh85** [rxhanson/Rectangle#876] [General]
When stage manager is active, if mouse moves to "far enough left", where the "far enough" parameter can be set in the Rectangle settings, then start snapping

Also, we need allow user to set the full screen size so that the stage manager dock can stay open
Instead of maximizing the application to fill the screen, only fill until the stage manager recent app doc

**Comment by rxhanson** [comment]
Sorry for the delay on this one. I'm not sure I understand exactly what you're looking for.

> When stage manager is active, if mouse moves to "far enough left", where the "far enough" parameter can be set in the Rectangle settings, then start snapping

Are you looking to adjust the snap area around the stage manager recent apps? This is something I have planned to look into but am waiting until closer to an official macOS release.

> Also, we need allow user to set the full screen size so that the stage manager dock can stay open
Instead of maximizing the application to fill the screen, only fill until the stage manager recent app doc

Which version of Rectangle are you using? The latest version of Rectangle does take into account the Stage Manager recent apps. Are you experiencing an issue with this?

**Reply by thloh85** [reply]
i think mean snapping. i think the more recent Rectangle already has the number of pixels for stage manager size, i just didn’t update it previously when testing thus i thought the feature wasn’t implemented
